test/module/irohad/torii/processor/transaction_processor_test.cpp

```diff
@@ -85,19 +83,19 @@ TEST_F(TransactionProcessorTest,
  * Transaction processor test case, when handling invalid transaction
  */
 TEST_F(TransactionProcessorTest,
-     TransactionProcessorWhereInvokeInvalidTransaction) {
-
+       TransactionProcessorWhereInvokeInvalidTransaction) {
   EXPECT_CALL(*pcs, propagate_transaction(_)).Times(0);
 
-  EXPECT_CALL(*validation, validate(A<const Transaction&>())).WillRepeatedly(Return(false));
-  EXPECT_CALL(*validation, validate(A<std::shared_ptr<const Query>>())).WillRepeatedly(Return(false));
+  EXPECT_CALL(*validation, validate(A<const Transaction &>()))
+      .WillRepeatedly(Return(false));
```

Comment:
where expect_call for Query?

Reply:
Transaction processor does not accept queries.

Reply:
Those comments, make me sad. Maybe we should conduct a meeting about architecture and responsibilities of key components?